### ISSUE TITLE
Certificate error ('not standards compliant'), after all

### ISSUE THREAD

**Issue by henkvantijen** [sahilph/umbrel-HTTPSizer#9]
At first everything seemed to work like described. Certificate gets blue checkmark in Keychain Access app [MacOS]. 
However, when I open an app from the Umbrel store, I get the message "not safe".  
When I click details, I get this dialog, that tells  certificate is 'not standards compliant'
<img width="688" alt="image" src="https://github.com/user-attachments/assets/0b4d39d3-1093-4daa-bed6-6c07c8fba273">

Even when I choose Always Trust and hit OK, the next time I get same warning.

**Comment by sahilph** [comment]
Yes Safari is known give this error even though its not supposed to for user-added Root CAs. [This](https://support.apple.com/en-us/102028) is the official Apple support Article about it. Check the last point in the notes

Nevertheless, since its our own self signed certificate you can ignore that error.
